src/main/java/com/home/MyWizardBot/MyWizardTelegramBot.java

```diff
@@ -0,0 +1,49 @@
+package com.home.MyWizardBot;
+
+import lombok.Setter;
+import org.telegram.telegrambots.bots.DefaultBotOptions;
+import org.telegram.telegrambots.bots.TelegramWebhookBot;
+import org.telegram.telegrambots.meta.api.methods.BotApiMethod;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+@Setter
+public class MyWizardTelegramBot extends TelegramWebhookBot {
+
+    private String webHookPath;
+    private String botUserName;
+    private String botToken;
+
+    @Override
+    public BotApiMethod<?> onWebhookUpdateReceived(Update update) {
+        // We check if the update has a message and the message has text
+        if (update.getMessage() != null && update.getMessage().hasText()) {
+            // Set variables
+            long chatID = update.getMessage().getChatId();
+
+            try {
+                // Create a message object object
+                execute(new SendMessage(chatID, "Hi" + update.getMessage().getText()));
+            }catch (TelegramApiException e) {
+                e.printStackTrace();
+            }
+        }
+        return null;
```

Comment:
Why did you decide to return `null`?

Reply:
overwriting the onWebhookUpdateReceived method
to return an object of class SendMessage